components/Header/Header.tsx

```diff
@@ -0,0 +1,124 @@
+import Image from 'next/image';
+import React, { useState, useEffect } from 'react';
+import { Menu, X } from 'react-feather';
+
+type MenuItem = {
+  text: string;
+  path?: string;
+};
+const defaultMenuItems: MenuItem[] = [
+  {
+    text: 'About us'
+  },
+  {
+    text: 'Press'
+  },
+  {
+    text: 'Careers'
+  },
+  {
+    text: 'Legal'
+  },
+  {
+    text: 'Support'
+  },
+  {
+    text: 'Contact'
+  },
+  {
+    text: 'Sitemap'
+  },
+  {
+    text: 'Cookie settings'
+  }
+];
+
+type TopMenuItem = {
+  text: string;
+  path?: string;
+};
+type HeaderParams = {
+  imageUrl?: string;
+  title?: string;
+  topMenuItems?: TopMenuItem[];
+  menuItems?: MenuItem[];
+};
+
+const defaultTopMenuItems: TopMenuItem[] = [
+  {
+    text: 'Home'
+  },
+  {
+    text: 'Products'
+  },
+  {
+    text: 'Resource'
+  },
+  {
+    text: 'Pricing'
+  }
+];
+export default function Header({
+  imageUrl = 'https://prototyprio.gumlet.io/strapi/20b5e1615519f85953d636ec6a5005ae.png?w=3840&q=75&format=webp&compress=true&dpr=2',
+  title = 'Untitled UI',
+  topMenuItems = defaultTopMenuItems,
+  menuItems = defaultMenuItems
+}: HeaderParams) {
+  const [isOpen, setIsOpen] = useState(true);
+
+  useEffect(() => {
+    if (typeof window !== 'undefined' && window.innerWidth < 768) {
+      setIsOpen(false);
+    }
+  }, []);
+  return (
+    <div className="md:h-auto h-screen flex md:p-4 p-0  gap-6 items-center flex-col md:flex-row">
+      <div className="flex border-b md:border-0 md:p-0 p-4  items-center justify-between md:w-fit w-full">
+        <div className="flex md:w-max items-center gap-x-3">
+          <Image alt="" src={imageUrl} width="45" height="45" />
+          <h2>{title}</h2>
+        </div>
+        <div className="flex md:hidden">
+          {isOpen ? (
+            <X onClick={() => setIsOpen(false)} />
+          ) : (
+            <Menu onClick={() => setIsOpen(true)} />
+          )}
+        </div>
+      </div>
+
+      {isOpen ? (
+        <>
+          <div className="flex  md:p-0 md:w-max p-4  grow w-full items-start md:items-center justify-start">
```

Comment:
There are formatting issues in className strings -double or triple spaces- here and there. Would be easy to fix after #40 i guess.

Reply:
yes, thanks. Prettier and eslint did not match when we were developing this component. So we added this to the "What is missed" field when opening the merge-request. we will update.

Reply:
Should we use the semantic button element or wait for the developed button element?

Reply:
nope. this is not about our button component. you need to use the native button element `<button>`